ambry-tools/src/main/java/com.github.ambry/store/CompactionVerifier.java

```diff
@@ -246,32 +246,22 @@ private CompactionVerifier(CompactionVerifierConfig verifierConfig, StoreConfig
     LOGGER.info("Delete ref time: {} ms", deleteRefTimeMs);
 
     MetricRegistry metricRegistry = new MetricRegistry();
-    AggregatedStoreMetrics aggregatedStoreMetrics = new AggregatedStoreMetrics(metricRegistry);
-    StoreMetrics srcMetrics = new StoreMetrics(verifierConfig.storeId + "-src", metricRegistry, aggregatedStoreMetrics);
-    StoreMetrics tgtMetrics = new StoreMetrics(verifierConfig.storeId + "-tgt", metricRegistry, aggregatedStoreMetrics);
+    StoreMetrics srcMetrics = new StoreMetrics("src", metricRegistry);
```

Comment:
Remind me about this verifier. Is it an independent tool?

Reply:
Yes it is.